cirq-core/cirq/transformers/measurement_transformers.py

```diff
@@ -139,6 +147,37 @@ def defer(op: 'cirq.Operation', _) -> 'cirq.OP_TREE':
     return circuit
 
 
+def _all_possible_datastore_states(
+    keys: Iterable['cirq.MeasurementKey'],
```

Comment:
Needs to be a set. It's not well-defined if there's a dupe. (i.e. condition is `m==m`.